types/chrome/index.d.ts

```diff
@@ -6742,6 +6742,159 @@ declare namespace chrome.printerProvider {
     export var onPrintRequested: PrintRequestedEvent;
 }
 
+////////////////////
+// Printing
+////////////////////
+/**
+ * Use the chrome.printing API to send print jobs to printers installed on Chromebook.
+
+ * Permissions: "printing"
+ * @platform ChromeOS only
+ * @since Chrome 81
+ */
+declare namespace chrome.printing {
+    export interface GetPrinterInfoResponse {
+        /** Printer capabilities in [CDD format](https://developers.google.com/cloud-print/docs/cdd#cdd-example). The property may be missing. */
+        capabilities?: object;
```

Comment:
The type `object` is generally discouraged. The type `object` represents any non-primitive type, which is way too broad. Looks like you're trying to make a type for a generic object, so I'd suggest using `{ [key: string]: unknown }`.

Reply:
I was hesitant to strictly follow the doc by putting `object` but I prefer your suggestion :)